broadcast/src/linked/signer/actor.rs

```diff
@@ -337,26 +352,25 @@ impl<
                     };
                     if let Err(err) = self.handle_ack(&ack).await {
                         warn!(?err, ?ack, "ack handle failed");
+                        guard.set(StatusValue::Failure);
                         continue;
                     }
+                    guard.set(StatusValue::Success);
                 },
 
                 // Handle completed verification futures.
                 maybe_verified = self.pending_verifies.next_completed() => {
                     let (context, digest, verify_result) = maybe_verified;
+                    self.metrics.verify.inc_with_bool(verify_result.is_ok());
```

Comment:
Since we typically match on anything that we use `inc_with_bool` with, I generally think we should just handle this manually (with explicit inc(), dec()).

Reply:
`inc_with_bool` will always increment, just as `Success` or `Failure` depending on the bool

Based on what you said, I think you assumed that it would either increment or not increment (?)

Removing this function might look like:
```
self.metrics.serve.inc(if result.is_ok() {StatusValue::Success} else {StatusValue::Failure});
```

Reply:
Okay removed `inc_with_bool`